### ISSUE TITLE
glusterfs.created force=True is not appending the force parameter

### ISSUE THREAD

**Issue by bogdanr** [saltstack/salt#28289]
I have this state file:

```
chgluster-create:
  glusterfs.created:
    - name: chgluster
    - bricks:
        - web1:/srv/gluster
        - web2:/srv/gluster
    - replica: 2
    - force: True
    - start: True
```
Running the state produces the following warning message and the force parameter isn't appended in salt 2015.8.1:
```
'force' is an invalid keyword argument for 'glusterfs.created'. If you were trying to pass additional data to be used in a template context, please populate 'context' with 'key: value' pairs. Your approach will work until Salt Carbon is out. Please update your state files.
```
According to [this](https://github.com/saltstack/salt/blob/2015.8/salt/states/glusterfs.py#L92) I understand that it should work. Am I missing something?

**Comment by jfindlay** [comment]
@bogdanr, thanks for the report.  Do you have any more log context for what you have posted?  I don't see `force` in the list of [reserved state keywords](https://github.com/saltstack/salt/blob/v2015.8.1/salt/state.py#L101), but it appears that this error is produced by `salt.utils.format_call` which is used internally by the state compiler.  Is it possible for you to also post the command you are using to run the state?

**Comment by bogdanr** [comment]
Seems to be fixed in 2015.8.1-122-gacff40c
